app/models/group.rb

```diff
@@ -571,6 +574,16 @@ def self.owner_of(groups, user)
     self.member_of(groups, user).where("gu.owner")
   end
 
+  def trigger_group_created_event
+    DiscourseEvent.trigger(:group_created, self)
```

Comment:
Hmm should we skip the trigger for automatic groups?

Reply:
We already have default `system` and `discobot` users along with [`user_created` event](https://github.com/discourse/discourse/blob/35745166b5a4d1f48b206192b21d8f8b44be67ef/app/models/user.rb#L1193-L1196) without any issues. So I think it won't be a problem.